fallback_detection/DetectingSlowFallbacks.jl

```diff
@@ -0,0 +1,44 @@
+# Title:  Variable Capture
+# Artist: Lyndon White (@oxinabox)
+# Format: Literate.jl script
+# License: MIT
+# ---
+
+# # What are we doing?
+#
+# We would like to discoure if we are hitting any slow cases for linear algebra operations.
+
+
+using LinearAlgebra
+using Cassette
+
+Cassette.@context GenericLogCtx
+
+function Cassette.prehook(
+    ::GenericLogCtx,
+    f::Union{
+        typeof(LinearAlgebra.generic_lufact!),
+        typeof(LinearAlgebra.generic_matmatmul),
+        typeof(LinearAlgebra.generic_matmatmul!),
+        typeof(LinearAlgebra.generic_matvecmul!),
+        typeof(LinearAlgebra.generic_mul!),
+        typeof(LinearAlgebra.generic_norm1),
+        typeof(LinearAlgebra.generic_norm2),
+        typeof(LinearAlgebra.generic_normInf),
+        typeof(LinearAlgebra.generic_normMinusInf),
+        typeof(LinearAlgebra.generic_normp),
```

Comment:
apparently `generic_norm` is not slow.

```suggestion
        ),
```